js/main.js

```diff
@@ -165,6 +165,23 @@ const popUpAppear = () => {
   });
 };
 
+const addData = (formName, formEmail, formMsg) => {
+  const myStorage = { formName, formEmail, formMsg };
+  localStorage.setItem('myStorage', JSON.stringify(myStorage));
+};
+
+form.addEventListener('input', () => {
```

Comment:
- [ ] When you start filling the form and refresh(reload) the page, the form should appear pre-filled with data you initially entered because it should be save to the local storage, your form is not behaving like that because the changes in the input fields are not saved the the local storage. Your have to add a <'change'> event listener to your form, to track any changes and save them to the local storage, such that, when you refresh or reload the page, it gets this information from the local storage and pre-fills the form. Please kindly fix that for it is the main requirement of this project. Please this [article](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event) will be helpful.